compiler/crates/relay-typegen/tests/generate_typescript/fixtures/conditional.expected

```diff
@@ -0,0 +1,36 @@
+==================================== INPUT ====================================
+fragment ConditionField on Node {
+  id @include(if: $condition)
+}
+
+fragment NestedCondition on Node {
+  ... @include(if: $condition) {
+    id
+  }
+}
+==================================== OUTPUT ===================================
+import { FragmentReference } from "relay-runtime";
+export type ConditionField$ref = FragmentReference & { _: "ConditionField$ref" };
+export type ConditionField$fragmentType = ConditionField$ref & { _: "ConditionField$fragmentType" };
+export type ConditionField = {
+  readonly id?: string,
+  readonly $refType: ConditionField$ref,
+};
+export type ConditionField$data = ConditionField;
+export type ConditionField$key = {
+  readonly $data?: ConditionField$data,
+  readonly $fragmentRefs: ConditionField$ref,
+};
+-------------------------------------------------------------------------------
+import { FragmentReference } from "relay-runtime";
```

Comment:
think though maybe this should `import type {}` no?

Reply:
Not a requirement, but definitely something to follow up with.

Reply:
@maraisr correct. It was a conscious decision to use this option since `import type` was a relatively recent addition to Typescript, and I wouldn't want to make this unusable for people on older versions.

Reply:
The fragment reference import _does_ currently use the `type` keyword, though.